src/fides/api/ops/models/application_config.py

```diff
@@ -82,3 +113,61 @@ def get_api_set_config(cls, db: Session) -> Dict[str, Any]:
         if config_record:
             return config_record.api_set
         return {}
+
+    @classmethod
+    def get_config_set(cls, db: Session) -> Dict[str, Any]:
+        """
+        Utility method to get the config_set config dict
+
+        An empty `dict` will be returned if no config settings have been set through config
+        """
+        config_record = db.query(cls).first()
+        if config_record:
+            return config_record.config_set
+        return {}
+
+    @classmethod
+    def update_api_set(
```

Comment:
Is there any way to reset "api_set"? That might be a nice feature - to just remove everything set by the API and revert to the default.  I found myself wanting to do that in testing.

Reply:
@pattisdr  great idea, i also felt that need a bit - seemed simple enough to add here so i added in https://github.com/ethyca/fides/pull/2551/commits/b053049e474d08de4dfc2e72a738d6d7fabf0135, let me know what you think

Reply:
looks great thank you